install/README.md

```diff
@@ -81,6 +81,7 @@ Manual installation instructions are available for [these distributions](#suppor
 |[Red Hat Enterprise Linux (RHEL)](rhel-installation-guide.md)      | 7               |
 |[SUSE Linux Enterprise Server (SLES)](sles-installation-guide.md)  | SLES 12 SP3     |
 |[Ubuntu](ubuntu-installation-guide.md)                             | 16.04, 18.04    |
+|[Debian](debian-installation-guide.md)                             | 9               |
```

Comment:
Nit: Could you move that line up so it is immediately below the CentOS line so the distro names are sorted alphabetically?

Reply:
done